tools/ota_e2e_tests/aws_ota_test/aws_ota_project.py

```diff
@@ -302,6 +304,55 @@ def setOtaBlockSize(self, blockSize):
             {' #define otaconfigMAX_NUM_BLOCKS_REQUEST': str(blockSize) + 'U'},
             os.path.join(self._projectRootDir, self._boardProjectPath, 'config_files', 'aws_ota_agent_config.h')
         )
+ 
+    def __insertTexts(self, prefix, texts, filePath):
```

Comment:
double leading underscore means reserved by system in python, use single leading underscore for private method.

Reply:
Originally the whole scripts was written in this way with double leading underscore. Just trying to keep consistency now :-)

Reply:
I see, let's correct the wrong consistency with a separate PR.